internal/pkg/action/operator_uninstall.go

```diff
@@ -86,45 +87,39 @@ func (u *OperatorUninstall) Run(ctx context.Context) error {
 		return err
 	}
 
-	if csv != nil {
-		// Ensure CustomResourceDefinitions are deleted next, so that the operator
-		// has a chance to handle CRs that have finalizers.
-		if u.DeleteCRDs {
-			crds := getCRDs(csv)
-			if err := u.deleteObjects(ctx, crds...); err != nil {
-				return err
-			}
-		}
-
-		// OLM puts an ownerref on every namespaced resource to the CSV,
-		// and an owner label on every cluster scoped resource. When CSV is deleted
-		// kube and olm gc will remove all the referenced resources.
-		if err := u.deleteObjects(ctx, csv); err != nil {
-			return err
-		}
+	operands, err := u.operandLister.Run(ctx, u.Package)
+	if err != nil {
+		return fmt.Errorf("listing operands associated with operator %s: %s", u.Package, err)
 	}
+	u.Logf("found %d operands associated with operator %s", len(operands.Items), u.Package)
+	u.Logf("operand deletion strategy: %s", u.OperandStrategy.String())
```

Comment:
I'd suggest removing these logs, and sticking to `<verb>ing <gvk> <name>/<namespace>` style logs to maintain consistency.

Eg.
- `deleting <gvk> <name>/<namespace>`
- `orphaning <gvk> <name>/<namespace>`